### DISCUSSION TITLE
New Server, having issues with lots of commands

### DISCUSSION THREAD

**Discussion by DonavynDeRaine** [Pryaxis/TShock#2110] [Q&A] [answered]
Hello everyone.  Before I begin I want to ensure everyone that I've done my share of research on the web to see if I can self-resolve the issue.  I'm out of ideas so now I turn to you all.

Scenario:

Just setup a new server for my game clan to use, and I'm having trouble adding someone to the admin group.  After logging in as the /owner I issued a /su to grant myself SuperUser rights.  I then attempted to add the user with the following syntax:

`/sudo /user group (username) admin`

Every attempt fails, telling me that the /user syntax isn't correct and to search for help for /user.

I've had a few other tech friends of mine look at the screen with me to see what I could be doing wrong, and we all end up scratching our collective heads on it.  When I look at the server-side information via Putty, the log file is HUGE, with mass entries in it such as the attached image to this post.

![image](https://user-images.githubusercontent.com/37151580/89701419-14c07b00-d8f4-11ea-8163-4e2a78128aa4.png)

`Bouncer / OnNewProjectile please report to tshock about this! normally this is a reject from Pax Empyrean 373 (sometimes 20)`

I will also add that he appears to be the only user who is experiencing the issue.  Pax's user account has been registered to the server and he has successfully logged into it.  His game-play is not interrupted at all on the user side, but this is what's happening on the server-side.

I'm running the server on a Debian Linux VM with sufficient resources dedicated to it.  Any and all help is greatly appreciated.

**Comment by DonavynDeRaine** [comment]
Another screen capture:

![Capture](https://user-images.githubusercontent.com/37151580/89701523-03c43980-d8f5-11ea-9f31-164a9671a212.PNG)

**Comment by hakusaro** [comment]
Can you try `user group username admin` from the console?

> I've had a few other tech friends of mine look at the screen with me to see what I could be doing wrong, and we all end up scratching our collective heads on it. When I look at the server-side information via Putty, the log file is HUGE, with mass entries in it such as the attached image to this post.

You can turn off debug logs in the config file (search for "DebugLogs" and set to `false`).
GitHub <notifications@github.com> wrote:
“Another screen capture:”

[Capture]
“ —

**Reply by DonavynDeRaine** [reply]
I've attempted to do that at the terminal as well, but did it again to be sure as well as capture another screen shot.  See below:

![Capture](https://user-images.githubusercontent.com/37151580/89735884-c52c9d00-da22-11ea-830b-36e585c03b7e.PNG)

**Comment by DonavynDeRaine** [comment]
It should also be mentioned that Pax is already part of the 'default' group, too.

**Comment by DonavynDeRaine** [comment]
Another thing that we're experiencing is when Pax lays down a teleporter, only he can see and use it.  Anyone else is then unable to place a teleporter gem because one already exists in the biom.

**Comment by DonavynDeRaine** [comment, marked as answer]
Seems that I may have stumbled upon the solution, would like to have feedback though in case it's still incorrect:

`/user group <my_account> superadmin`

Previously my account was listed in the 'owner' group, which I thought was the 'owner of the server'.  Looking at it more closely it appears that it was a GROUP owner instead.  Once I moved myself to super admin, I was able to issue the following command at the terminal level:

`/user group "Pax Empyrean" trustedadmin`

I first attempted to add him to just the 'admin' group, but he didn't have the ability to move NPC's and such.  Moving him to the trustedadmin group seems to allow him to have the ability to function like self-hosted server game.

One more question though, where can I see a list of permissions that each group has assigned to it, so I can modify that accordingly?

**Reply by bartico6** [reply]
/group listperms <group> afaik.
Check /group help, it's either listperm or listperms

**Reply by DonavynDeRaine** [reply]
Perfect, it was:

`/group listperm <group> <page>`

For trustedadmin there seems to be 8 pages of permissions that they have.  Thanks so much for the assist with that. =)
